src/BinSkim.Rules/EnableSpectreMitigations.cs

```diff
@@ -40,13 +41,9 @@ protected override IEnumerable<string> FormatIds
             {
                 return new string[] {
                     nameof(RuleResources.BA2024_Error),
-                    nameof(RuleResources.BA2024_Error_BuildWithSpectreMitigation_BadCompilerVersion),
-                    nameof(RuleResources.BA2024_Error_BuiildWithSpectreMitigation_UnrecognizedCompiler),
-                    nameof(RuleResources.BA2024_Error_BuildWithSpectreMitigation_SpectreMitigationMissing),
-                    nameof(RuleResources.BA2024_Error_BuildWithSpectreMitigation_SpectreMitigationDisabled),
```

Comment:
As per discussions - I do not feel we should not be disabling checks for MASM, version and other things in this rule just because they are not supported in the rest of the tool's checks.

I know there are feelings about this being too aggressive etc, but given the severity of Spectre we should make this an opt-in check and provide all the information.

If users can specify modules they assert to have checked and been clean we have user performable actions for every error and warning.

Reply:
ok. will address this in a future change.